pkgs/development/libraries/science/astronomy/indilib/indi-3rdparty.nix

```
@@ -37,16 +38,21 @@ stdenv.mkDerivation rec {
     firmware
   ];
 
-  postPatch = ''
+    
+
+  patchPhase = ''
     for f in indi-qsi/CMakeLists.txt \
              indi-dsi/CMakeLists.txt \
              indi-armadillo-platypus/CMakeLists.txt \
```

```suggestion
    indi-dsi/CMakeLists.txt \
    indi-armadillo-platypus/CMakeLists.txt \
```
please fix the spacing. We are nowhere using 8 spaces